### ISSUE TITLE
Missing Tasks

### ISSUE THREAD

**Issue by jaseweston** [facebookresearch/ParlAI#492]
This is a list of tasks not yet in ParlAI that would be great to have. Feel free to add more to the list also!  We will remove individual items when they are done.

    * Multi-hop Reading Comprehension : https://arxiv.org/pdf/1710.06481.pdf
    * SearchQA https://github.com/nyu-dl/SearchQA
    * Twitter data used in e.g. https://arxiv.org/pdf/1603.06155v2.pdf
    * Who Did What (cloze qa): https://arxiv.org/abs/1608.05457 
    * NewsQA https://datasets.maluuba.com/NewsQA

**Comment by apsdehal** [comment]
@jaseweston How about AQuA dataset by deepmind for algebraic questions? https://github.com/deepmind/AQuA
I can add support for this if it is acceptable.

I can possibly work on others too.

**Comment by jaseweston** [comment]
yes, sure -- it would be great if you can add it (and any others you want)!

**Comment by dfcf93** [comment]
How about the Dureader dataset by Baidu for MRC? https://github.com/baidu/DuReader

**Comment by jaseweston** [comment]
sure it would be great if you can add it!

**Comment by BenjaminWinter** [comment]
I would add HotpotQA https://hotpotqa.github.io/, which my group are currently working on getting into ParlAI

**Comment by jaseweston** [comment]
> I would add HotpotQA https://hotpotqa.github.io/, which my group are currently working on getting into ParlAI

awesome!

**Comment by ricsinaruto** [comment]
I am commenting here to let interested people know that I have a working DailyDialog implementation running on this fork: https://github.com/Mrpatekful/ParlAI/tree/dialogwae.

I will submit a PR as soon as I get around to clean the code and run some tests. I have to note though that in my implementation I only use the chat dataset without any annotations, so I guess it's not a full implementation of DailyDialog, but I would be happy to collaborate if someone were up to it. I did implement two types of tasks though: a single-turn task, and a task where all dialog history is used, for agents that can handle multiple utterances.

**Comment by abisee** [comment]
There's a large new grounded dialogue dataset that came out last month:
**Conversing by Reading: Contentful Neural Conversation with On-demand Machine Reading**
https://arxiv.org/pdf/1906.02738.pdf
Might be useful?

**Comment by jaseweston** [comment]
Yes, it was already in our list..
